bi_sql_editor/models/bi_sql_view.py

```diff
@@ -639,7 +615,7 @@ def _check_execution(self):
         field_ids = []
         for column in columns:
             existing_field = self.bi_sql_view_field_ids.filtered(
-                lambda x: x.name == column[1]
+                lambda x: x.name == column[1]  # noqa: B023
```

Comment:
Why this line. I dont understand B2023. What is dangerous here ?

Reply:
yep. "**B2023**" looks very dangerous. luckily we got `B023` here instead, and, it's a ruff failed code. any idea to fix it is welcome.

Reply:
> "B2023" looks very dangerous.

sorry, it's a typo.

> any idea to fix it is welcome.

Let's discuss here https://github.com/OCA/oca-addons-repo-template/issues/241

Reply:
It should be defined like this
`lambda x, c=column: x.name == c[1]`

Reply:
Thanks @rven ! 

@guyenminhchien could you use this syntax ? 

Thanks !

Reply:
updated